kanvas/src/androidMain/kotlin/KanvasView.kt

```diff
@@ -33,5 +32,5 @@ abstract class KanvasView(
      * Unlike [View.onDraw], you should use [Canvas.width] and [Canvas.height] instead of the view's
```

Comment:
This is supposed to refer to `Kanvas`, right?

```suggestion
     * Unlike [View.onDraw], you should use [Kanvas.width] and [Kanvas.height] instead of the view's
```